snmp/assets/dashboards/netflow_monitoring.json

```diff
@@ -513,7 +511,10 @@
                                     "display_type": "line",
                                     "formulas": [
                                         {
-                                            "formula": "(query1 * 8)"
+                                            "formula": "query1"
+                                        },
+                                        {
+                                            "formula": "query1 * 8"
```

Comment:
This widget is in bits:
1. Does it make sense to have it in bits if all others are in bytes?
2. Why do we have both `query1` and `query1 * 8`, are we showing both units?
3. We seem to be using an `avg` aggregation. Should we change to `sum`?

Reply:
Good catch for the sum and having both queries 👍 
For the unit, Total Traffic from Top 10 Devices and Traffic by Interface are the only widgets we have in bits. I was not sure if this was a product decision @angelinajin , but since it was specified in the title, I didn't change it. I don't have a strong opinion about this.

Reply:
Make sense, let's leave it like that it's explicit what's being done anyway 👍